### PR TITLE
Remove pandas and numpy

### PR DESCRIPTION
**Pull request by sevein**
This pull request removes a few dependencies that are unused or
underused: pandas, numpy and django-model-utils.

pandas is great but it's definitely underused in Archivematica while
taking near [half of the space] used by our virtual environments plus
adding some complexity to our builds that we don't have capacity to
tackle right now.

Connects to archivematica/Issues#392.

[half of the space]: https://gist.github.com/sevein/43bb237f7139e22c9f16e5bfddb69295

### CODE REVIEW

File: src/dashboard/src/components/archival_storage/tests/test_archival_storage.py [hunk 1]
```diff
@@ -43,6 +44,16 @@
...
 
 
+@pytest.fixture
+def amsetup(db):
+    setting, _ = DashboardSetting.objects.get_or_create(
+        name="dashboard_uuid", defaults={"value": str(uuid.uuid4())}
+    )
+    return {
+        "uuid": setting.value,
+    }
```
Comment (sevein) [inline src/dashboard/src/components/archival_storage/tests/test_archival_storage.py:54]: I'm hoping this fixture could eventually be exported by conftest and contain other installation attributes.

File: src/dashboard/src/components/archival_storage/views.py [hunk 2]
```diff
@@ -189,109 +188,89 @@ def get_es_property_from_column_index(index, file_mode):
...
 
 
-def _ordered_dict_from_es_fields():
-    """Ordered dict from es fields
+# Ordered dict from es fields.
+#
+# Archivematica code currently uses a mix of snake case, camel case,
+# and other inconsistent variants of both for its Elasticsearch
+# field names. Here we create a mapping to enable their output to
+# something nice to read. We can also use this function to guarantee
```
Comment (replaceafill) [inline src/dashboard/src/components/archival_storage/views.py]:
```suggestion
# something nice to read. We can also use this dictionary to guarantee
```

File: src/dashboard/src/components/archival_storage/views.py [hunk 3]
```diff
@@ -189,109 +188,89 @@ def get_es_property_from_column_index(index, file_mode):
...
+        try:
+            created = make_aware(
+                datetime.fromtimestamp(row[es.ES_FIELD_CREATED]),
+                timezone=current_timezone,
+            )
+        except TypeError:
+            created = row[es.ES_FIELD_CREATED]
+        row[es.ES_FIELD_CREATED] = created
+
+        def encode(item):
```
Comment (replaceafill) [inline src/dashboard/src/components/archival_storage/views.py]: Move this function out of the `for` loop?

**Review by replaceafill — APPROVED** [review]
LGTM :+1:

**Pull request merged**